projects/swimlane/ngx-ui/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## HEAD (unreleased)
 
+## 45.0.1 (2023-7-11)
+
+- Fix (`json-editor`): When we added a datetime input you weren't able set a value because the model value didn't have the correct format.
```

Comment:
Let's simplify the description of the fix to the following:

`Datetime input now uses the correct format`

Reply:
Done!